server/templates/login.html

```diff
@@ -33,6 +33,11 @@ <h2 class="form-signin-heading">Welcome to Svarog</h2>
 <p>Welcome, {{ user.username }}!</p>
 <p>Your user-id is {{ user.id }}.</p>
 <p>Your role is {{ user.role.name }}.</p>
+<p>You are the owner of the following {{ stations|length }} station(s):
+{% for s in stations %}
+    <a class="param" href="/station/{{ s.station_id }}">{{s.name}} (id: {{s.station_id}})</a>
```

Comment:
Probably the user shouldn't see our internal ID (?).

Reply:
The station id is visible in the URL anyway, so it's not exactly secret. But I agree. It shouldn't be visible. Removed.